params/config.go

```diff
@@ -769,6 +769,10 @@ func (c *ChainConfig) IsCel2(time uint64) bool {
 	return isTimestampForked(c.Cel2Time, time)
 }
 
+func (c *ChainConfig) IsPreCel2(time uint64) bool {
+	return !c.IsCel2(time)
```

Comment:
This doesn't look right to me, IsPreCel2 will return true in situations where the cel2 fork was not set

Reply:
Yes, Optimism has the `IsOptimism` check in addition. For us, it should probably be something like `c.Cel2Time != nil && !c.IsCel2(time)`

Reply:
I'm just testing this now